lua/yabs/defaults/output/quickfix.lua

```diff
@@ -1,13 +1,11 @@
 local open_on_run
 
-local open = false
 local function on_read(lines)
     for index, line in ipairs(lines) do
         if index == #lines and line == "" then goto continue end
 
         vim.fn.setqflist({}, "a", {lines = {line}})
-        if not open and open_on_run == "auto" then
-            open = true
+        if open_on_run == "auto" then
             vim.cmd("bot copen")
             vim.cmd("wincmd p")
```

Comment:
Nitpick: Use `vim.api.nvim_command` instead, it works faster. See [this](https://github.com/hrsh7th/nvim-cmp/pull/168#discussion_r703072501) comment for details.

Reply:
I’ll look into this. Thanks for the suggestion!